software/trailbase.yml

```
@@ -0,0 +1,12 @@
+name: TrailBase
+website_url: https://trailbase.io/
+description: Open, sub-millisecond, single-executable FireBase alternative with type-safe REST & realtime APIs, built-in JS/TS runtime, auth & admin UI built on Rust, SQLite & V8.
```

```suggestion
description: Open, sub-millisecond, single-executable FireBase alternative with type-safe REST & realtime APIs.
```

Not relevant, short descriptions are better.

---

I can see the point about succinctness, I'm just not sure about the not relevant part. "type-safe REST & realtime APIs" feels like an arbitrary feature subset. What did you have in mind?

---

Hi @ignatz the main point is, the tech stack used to build/write the software is best left out of the description.

> arbitrary feature subset

It is common to list a few core features in the description - since "type-safe REST & realtime APIs" is also part of the official description on the website and the github project, I think we should also mention them here.

---

That makes sense. I dropped the trailing tech stack part and kept all the features. Thanks and PTAL